frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -1090,15 +1189,26 @@ function ContextMenuInner(propsWithInputRef: Props) {
       );
     }
 
-    if (segmentIdAtPosition > 0) {
+    if (segmentIdAtPosition > 0 || maybeClickedMeshId != null) {
+      const segmentId = maybeClickedMeshId ? maybeClickedMeshId : segmentIdAtPosition;
       infoRows.push(
         <InfoMenuItem key="copy-cell">
           <div className="cell-context-icon" />
-          Segment ID: {`${segmentIdAtPosition}`}{" "}
-          {copyIconWithTooltip(segmentIdAtPosition, "Copy Segment ID")}
+          Segment ID: {`${segmentId}`} {copyIconWithTooltip(segmentId, "Copy Segment ID")}
         </InfoMenuItem>,
       );
     }
+    if (segments != null && maybeClickedMeshId != null) {
+      const segmentName = getNameOfSegment(segments.get(maybeClickedMeshId));
+      if (segmentName != null) {
+        infoRows.push(
+          <InfoMenuItem key="copy-cell">
+            <div className="cell-context-icon" />
+            Segment Name: {segmentName}
```

Comment:
I am unsure how to indicate that this menu field displays the segment's name. First, I wrote `Segment Name:` as a prefix. But this seems to be too long IMO. Therefore I replayed it with a name tag icon. Does this make sense? Do you have a better suggestion?